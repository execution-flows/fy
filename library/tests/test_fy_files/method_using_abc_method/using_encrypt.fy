method greet using encrypt:
    with method encrypt

    def -> None:
        print(self._encrypt("Hello World!"))
