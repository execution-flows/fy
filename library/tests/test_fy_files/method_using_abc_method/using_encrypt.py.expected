import abc

from test_fy_files.method_using_abc_method.abc import Encrypt_MethodMixin_ABC


class Greet_UsingEncrypt_MethodMixin(
    # Method_mixins
    Encrypt_MethodMixin_ABC,
    abc.ABC,
):
    def _greet(self) -> None:
        print(self._encrypt("Hello World!"))
